staging/src/k8s.io/apiserver/pkg/endpoints/handlers/rest.go

```diff
@@ -52,6 +53,8 @@ type RequestScope struct {
 	Typer           runtime.ObjectTyper
 	UnsafeConvertor runtime.ObjectConvertor
 
+	OpenAPIGetter openapi.Getter
```

Comment:
So, it would be great if this struct bound it to a particular resource instead of giving access to the entire thing--I gave an example in another comment, hopefully that made sense.